### ISSUE TITLE
Remove Hobbs gauge

### ISSUE THREAD

**Issue by gilbertohasnofb** [Harald67/c150#8]
I would suggest to remove the Hobbs gauge given that the RPM gauge also displays the engine time (note how they even have the same value):

![c150-0](https://cloud.githubusercontent.com/assets/5700795/17267405/9587ba1e-5608-11e6-9ac9-b4141d5b3716.jpg)

Maybe we could add an EGT gauge there? See: http://www.jacobsflightservices.com/cms-assets/images/658174.20150422113524.jpg

**Comment by Harald67** [comment]
The two Hobbs gauges are not counting the same thing.
The one on the right is counting as soon as the engine is started.
The one on the left is counting based on the engine rpm (and the reference rpm value is probably wrong). In practice they have similar values except if you spend a lot of time taxiing.
We could probably put an EGT here (but frankly I think you found the only c150 panel with an egt), and add an option to display the egt or the hobbs. 
I wanted to make a dialog to configure a bit the panel (for example to use the light model of the arc radios or event use other brands of radios), but not for this release.

**Comment by gilbertohasnofb** [comment]
I see.

> (but frankly I think you found the only c150 panel with an egt)

That's funny, that was one of the first images I have found online. But after you mention it, I did realise that very often they have the Hobbs meter there. I have also seen once or twice an ADF gauge at that position.

> I wanted to make a dialog to configure a bit the panel (for example to use the light model of the arc radios or event use other brands of radios), but not for this release.

Yes, that's a nice idea but it's not a priority. My point was that I thought both the Hobbs and the tachometer counter were showing the same value, but since they are not and since you find configuration in several (or even most!) Cessnas 150 in real life, then I'd say we close this issue.
